test.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
we don't want to check this one in.

Reply:
It is there in .gitignore. Not sure why it has appeared 😕

Reply:
i think you overrode the .gitignore at some point with an explicit `git add test.html`.

luckily you can `git rm test.html` too.